extensions/wikia/LanguageWikisIndex/LanguageWikisIndexController.class.php

```diff
@@ -8,11 +8,29 @@ public function __construct() {
 	}
 
 	public function index() {
+		global $wgIncludeClosedWikiHandler;
 
 		$this->specialPage->setHeaders();
 
-		$this->response->setTemplateEngine( \WikiaResponse::TEMPLATE_ENGINE_MUSTACHE );
+		Wikia::addAssetsToOutput( 'language_wikis_index_scss' );
 
+		$this->setVal( 'langWikis', WikiFactory::getLanguageWikis() );
+
+		if ( !empty( $wgIncludeClosedWikiHandler ) ) {
```

Comment:
Cross-extension dependency. Maybe call a hook here and overwrite the into message in a hook listener defined in `closedWikiHandler`?

Reply:
I thought this global might be cheap since it's set in WFL rather than an extension, but rather than using hooks and having that flow complexity, I think we should just check if the domains city ID is not 3 here (or check `WikiFactory::isActive`). What do you think?